package.json

```diff
@@ -82,7 +84,8 @@
     "svgo-loader": "^1.1.2",
     "url-loader": "^0.5.8",
     "webpack": "^3.0.0",
-    "webpack-dev-server": "^2.5.0"
+    "webpack-dev-server": "^2.5.0",
+    "snyk": "^1.41.1"
```

Comment:
Shame it's not in alphabetical order—this will end up changing in the next update to package.json, since npm does it for you. Might update this manually when I get a sec.

Reply:
Also realised this is in dependencies, not dev dependencies. Consumers don't need this at all. Might need to jump into this branch and clean it up.